test/e2e/adminapi_cluster_getlogs.go

```diff
@@ -46,21 +46,21 @@ func testGetPodLogsOK(ctx context.Context, containerName, podName, namespace str
 
 	By("creating a test pod in openshift-azure-operator namespace with some known logs")
 	pod := mockPod(containerName, podName, namespace, expectedLog)
-	pod = CreateK8sObjectWithRetry(
+	CreateK8sObjectWithRetry(
 		ctx, clients.Kubernetes.CoreV1().Pods(namespace).Create, pod, metav1.CreateOptions{},
 	)
 
 	defer func() {
 		By("deleting the test pod")
 		DeleteK8sObjectWithRetry(
-			ctx, clients.Kubernetes.CoreV1().Pods(namespace).Delete, pod.Name, metav1.DeleteOptions{},
+			ctx, clients.Kubernetes.CoreV1().Pods(namespace).Delete, podName, metav1.DeleteOptions{},
```

Comment:
I'm at a loss for how the const defined inside a different function would be made available to this func?